queue/src/fiskaltrust.Middleware.Queue.AzureTableStorage/fiskaltrust.Middleware.Queue.AzureTableStorage.csproj

```diff
@@ -58,4 +50,8 @@
             <PackageCopyToOutput>true</PackageCopyToOutput>
         </Content>
     </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Update="fiskaltrust.storage" Version="1.3.66" />
```

Comment:
version is defined in this file
https://github.com/fiskaltrust/middleware/blob/main/queue/test/Directory.Build.props